graphs/LineGraph/ToolTip/index.js

```diff
@@ -14,7 +14,8 @@ module.exports = React.createClass({
     show: React.PropTypes.bool,
     data: React.PropTypes.object,
     label: React.PropTypes.string,
-    isFlipOver: React.PropTypes.bool
+    isFlipOver: React.PropTypes.bool,
```

Comment:
What does this do? I see that it is used, but I am not sure what it does. Not in the documentation